### DISCUSSION TITLE
21.6.5

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1398] [General]
## Releases Notes for 21.6.5
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.5/draw.io-21.6.5-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.5/draw.io-21.6.5-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.5/draw.io-universal-21.6.5.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.5/drawio-amd64-21.6.5.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.5/drawio-x86_64-21.6.5.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.5/drawio-x86_64-21.6.5.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 21.6.5](https://github.com/jgraph/drawio/blob/v21.6.5/ChangeLog). All changes from 21.6.2 to 21.6.5 are added in this build.
- Add ARM64 linux architectures. #1140 
- Reverts dark mode to legacy implementation by default. #1391 

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v21.6.5'>21.6.5</a>.</em>

**Comment by EBMBA** [comment]
Hello, 

When will the version available from the Windows Store be updated please? The current version is the 21.6.1 and it have a CVE (CVE-2023-3973). 

Thanks

**Reply by davidjgraph** [reply]
The desktop build is not vulnerable to 3973.

**Reply by EBMBA** [reply]
Ok thanks 

Defender triggers the v26.6.1 as vulnerable to the CVE-2023-3973

**Reply by davidjgraph** [reply]
That should be in the store now.

**Comment by basvandrongelen** [comment]
The chrome version isn't supported anymore. for more info, see https://support.google.com/chrome/thread/174381169?visit_id=638266438247875014-3156671649&p=chrome_app_deprecation&rd=1

**Reply by davidjgraph** [reply]
I was about to say "but not for ChromeOS", but no "January 2025: End support for Chrome Apps on ChromeOS.". What is wrong with this company. They need to just kill everything except search, since that's what's going to happen....
